lib/NonlinearSolveSciPy/src/NonlinearSolveSciPy.jl

```diff
@@ -0,0 +1,209 @@
+module NonlinearSolveSciPy
+
+using ConcreteStructs: @concrete
+using Reexport: @reexport
+
+using PythonCall: pyimport, pyfunc
+const scipy_optimize = try
+    pyimport("scipy.optimize")
+catch
+    nothing
+end
```

Comment:
See https://juliapy.github.io/PythonCall.jl/stable/pythoncall/#Precompilation on how to import python packages in the global scope correctly